external-types/arrayifyStream.d.ts

```diff
@@ -0,0 +1,6 @@
+declare module 'arrayify-stream' {
```

Comment:
Can these external types be moved to https://github.com/DefinitelyTyped/DefinitelyTyped/ ?

Reply:
Will have a look at the definitely typed stuff